components/tag-list/tag-list.js

```diff
@@ -0,0 +1,96 @@
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { ArrowKeysMixin } from '../../mixins/arrow-keys-mixin.js';
+import { ifDefined } from 'lit-html/directives/if-defined.js';
+
+const ITEM_SPACING = 6;
+
+class TagList extends ArrowKeysMixin(LitElement) {
+
+	static get properties() {
+		return {
+			/**
+			 * REQUIRED: A description of the tag list for additional accessibility context
+			 * @type {string}
+			 */
+			description: { type: String },
+			/**
+			 * @ignore
+			 */
+			role: { type: String, reflect: true },
+		};
+	}
+
+	static get styles() {
+		return css`
+			:host {
+				display: block;
+			}
+			:host([hidden]) {
+				display: none;
+			}
+			.tag-list-container {
+				display: flex;
+				flex-wrap: wrap;
+				margin: -${ITEM_SPACING}px -${ITEM_SPACING}px 0 0;
```

Comment:
Did this work as expected?  I don't think using non `css` expressions is recommended here.  Perhaps take a peek at [this section of the Lit doc](https://lit.dev/docs/components/styles/#expressions) - perhaps use a CSS variable instead.

Reply:
Odd, so this does work totally fine and Lit lets me do it.  But if I change it to string instead of a number, it gets mad.  I can wrap it in a css literal to be safe though.  I didn't use a variable because I didn't want people to be able to override it, this is just for ease of a developer updating it later (if needed).

Reply:
Just going to remove having this as a variable.  Our style linter doesn't allow for what the Lit docs recommends - it looks at all `css` tags and checks for valid css (meaning something like `whatever { color: red; }`, not `red`).